### PR TITLE
configurations

### PR DESCRIPTION
**Pull request by BnetButter**
Run file directly without having to update table. Tweaked logger so that the root logger receives all log events.

### CODE REVIEW

File: autonomy.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+def main():
+    logger.info("hello")
+    
+
```
Comment (BnetButter) [inline autonomy.py:8]: Set up callable placeholder for autonomy main.

File: example.py [hunk 2]
```diff
@@ -7,19 +7,17 @@ def main() -> None:
...
     bearing, distance = algorithms.geomath.haversine(37.951424, -91.768959, 37.951524, -91.768100)
 
     # Test sending RoveComm packets
     packet = core.RoveCommPacket(200, 'b', (1, 2), '')
     packet.SetIp('127.0.0.1')
     core.rovecomm_node.write(packet)
 
-    logger.info("Calculated bearing: %s", bearing)
-    logger.info("Calculate distance: %s", distance)
+    logger.info(f"Calculated bearing: {bearing}")
```
Comment (BnetButter) [inline example.py:19]: F-string interpolation for readability

File: run.py [hunk 3]
```diff
@@ -1,73 +1,89 @@
...
+def setup_logger(level) -> logging.Logger:
     '''
     Sets up the logger used in the autonomy project with appropriate
     handlers and formatting
 
     Returns
     -------
 
-        Logger: Autonomy_Logger set up for console and file logging
+        Logger: root set up for console and file logging
```
Comment (BnetButter) [inline run.py:16]: We basically only want to do log setup on the root logger.

File: run.py [hunk 4]
```diff
@@ -1,73 +1,89 @@
...
     '''
 
-    # logging file
-    logger = logging.getLogger('Autonomy_Logger')
+    logger = logging.getLogger()
```
Comment (BnetButter) [inline run.py:19]: Get the root logger instead of 'Autonomy_Logger'

File: run.py [hunk 5]
```diff
@@ -1,73 +1,89 @@
 import argparse, logging
+import logging.config
+import importlib
 import core
-# Import files that should be calleable from run.py here
-import example
+import os
 
-# Map all calleable file names to their respective mains here
-FUNCTIONS = {'example.py': example.main}
```
Comment (BnetButter) [inline run.py:7]: No longer needed... see bottom of the page

File: run.py [hunk 6]
```diff
@@ -1,73 +1,89 @@
...
+    core.rovecomm_node = core.RoveCommEthernetUdp()
 
-        try:
-            logger.info("Calling module: %s", args.file)
-            func()
-        except Exception as e:
-            logger.exception("Tried to run: %s had exception %s", args.file, e)
+    try:
+        # Remove .py and directly import module
+        module = importlib.import_module(os.path.splitext(args.file)[0])
```
Comment (BnetButter) [inline run.py:73]: Import whatever `.py` file as long as it defines `main()`

**Comment by EliVerbrugge** [comment]
As this is minor in scope and just fixes some existing issues, I'm going to go ahead and merge this.

**Pull request merged**